db/data/regions/KR/en.yml

```diff
@@ -5,10 +5,12 @@ en:
       addresses:
         zip:
           errors:
-            invalid: Enter a valid 5-digit postal code for South Korea
+            invalid_instructional: Enter a valid 5-digit postal code for South Korea
```

Comment:
Same comment as with Israel above, is it necessary to specify if they're _always_ 5 digits?